<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="CSStatu">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="autoScroll">
            <bool>true</bool>
           </property>
           <row>
            <property name="text">
             <string>Server</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>C/S连接状态</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>收发包状态</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string/>
            </property>
           </row>
           <row>
            <property name="text">
             <string>错误监控</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>IP</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Port</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>在线数</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>断线数</string>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>传输速度 (kb/s)</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>发包量</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string>丢包量</string>
            </property>
           </item>
           <item row="3" column="2">
            <property name="text">
             <string>发包速度 (pcs/s)</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>错误编码</string>
            </property>
           </item>
           <item row="5" column="1">
            <property name="text">
             <string>异常原因</string>
            </property>
           </item>
           <item row="5" column="2">
            <property name="text">
             <string>...</string>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTextEdit" name="s_ip">
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAsNeeded</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="s_port"/>
               </item>
               <item>
                <widget class="QTextEdit" name="s_pkg"/>
               </item>
               <item>
                <widget class="QTextEdit" name="s_socketsCount"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>目标IP</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_1">
                 <property name="text">
                  <string>目标Port</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>包体文本</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>申请连接数</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>心跳包</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>暂停发包</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>随机短线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="EntablConn">
              <property name="text">
               <string>建立连接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ShutCon">
              <property name="text">
               <string>重置连接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BeginSend">
              <property name="text">
               <string>并发测试</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
